devspace.yaml

```
@@ -263,7 +272,13 @@ deployments:
               - name: STALKER_VERSION
                 value: ${STALKER_VERSION}
 
-            image: jobs-manager
+            volumeMounts:
+              - containerPath: /certs
+                volume:
+                  name: certs
+                  readOnly: false
```

pourquoi readonly? Mon impression c'est que c'est pas necessaire d'ecrire dedans?
```suggestion
                  readOnly: true
```
Ce meme commentaire va etre la une couple de fois